src/components/accountTransactions/index.js

```diff
@@ -33,21 +51,27 @@ class accountTransactions extends React.Component {
         <Transactions
           history={this.props.history}
           address={this.props.match.params.address}
-          balance={this.props.balance} />
+          balance={this.props.balance}
+          delegate={this.props.delegate} />
       </div>
     </div>;
   }
 }
 
 const mapStateToProps = state => ({
-  delegateUsername: state.account.delegate ? state.account.delegate.username : null,
+  delegateUsername: (state.account && state.account.delegate)
+    ? state.account.delegate.username : null,
   balance: state.transactions.account ? state.transactions.account.balance : null,
   publicKey: state.account.publicKey,
   activePeer: state.peers.data,
   notLoading: state.loading.length === 0,
+  peers: state.peers,
+  delegate: state.account && state.account.delegate,
 });
 
 const mapDispatchToProps = dispatch => ({
+  accountVotersFetched: data => dispatch(accountVotersFetched(data)),
+  accountVotesFetched: data => dispatch(accountVotesFetched(data)),
```

Comment:
https://jenkins.lisk.io/job/lisk-hub/job/PR-765/20/cobertura/src_components_accountTransactions/index_js/
These two lines not being covered tell me that integration tests are missing. If too much work, integration tests can be a separate PR.